### DISCUSSION TITLE
Institutional Client Onboarding - Entity Validation (Ecosystem Grants)

### DISCUSSION THREAD

**Discussion by EFSG11** [crypto-org-chain/chain-main#764] [Ideas]
**Impact**

Crypto.com struggles with institutional onboarding of entities, especially across different geographies - with the average onboarding process taking 5-8 weeks depending on the size and jurisdiction of the entity. Over 40% of entities drop out of the onboarding process due to length of time, or intrusive information requests - this can pose a significant cost to the future growth of the community. The process is lengthy as Client Onboarding programs struggle to complete the process of Entity Validation - matching incorporation documents to primary public authoritative sources. This is a challenge as manual labour is employed to check entity details across multiple databases in differing jurisdictions and languages.  The problem is further amplified when trying to maintain this data post onboarding for any KYC/B refreshes. 

My team have developed a Proof of Concept that will utilize NLP sequencing to aggregate - in partnership with underlying public authoritative sources (e.g. business registries) - legal entity information, which will then be validated by a community consensus mechanism. This will allow legal entity information (and the proof of validation) to be tokenized and stored on a distributed ledger. This will increase the speed of Entity Validation requests, and in turn Client Onboarding lifecycles. We hope to link this Proof of Validation to other Digital IDs utilized by individuals to allow them to onboard themselves and their entities easier into the ecosystem. The solution primarily solves the speed of existing Entity Validation and the inherent consensus mechanism provides trust and maintenance to the data sourced. 

**Execution**

 We are best suited to tackle this problem as Junhua (CTO) has a pHD in Artificial Intelligence, specialising in NLP sequencing which is required for aggregating the information from contributing sources. Only a limited number of jurisdictional registries have available APIs. Complimenting this, John (CEO) was most previously the Partnerships Director for Asia with the London Stock Exchange and understands data distribution agreements and has a network with the underlying sources. We are of course open to the community to help us build this project to its completion!

**Timeline**
From July, we expect it reasonably to take 6 months to establish an MVP with the an initial supporting ecosystem, and early adopting primary source contributors that have APIs ready to aggregate information from in regards to legal entities. 

As less than 15% of primary public authoritative sources have robust data systems for their entity registries, and we need partnerships that license and facilitate utilizing NLP to extract the information - we expect at least a year further to bring all underlying sources for jurisdictions onboard.

This project should align with Phase IV & V of Crypto.com Chain. 

**Budget**

We expect this project to require USD 300,000 to see through to completion over the course of 18 months.

**Details**

Contact - John Gallagher (john@entrepreneurfirst.co)
Team/Company Name - Entity Chain (supported by Entrepreneur First)

**Comment by aw126** [comment]
Hi John, thanks for your interest in our ecosystem grant program. 
Your proposal has been received - we will forward the details to the relevant team who will work on your application.
